app/views/fields/select/_index.html.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+# Select Index Partial
+
+This partial renders a selectable text attribute,
+to be displayed on a resource's index page.
+
+## Local variables:
+
+- `field`:
+  An instance of [Administrate::Field::Select][1].
+  A wrapper around the attribute pulled from the database.
+
+[1]: http://www.rubydoc.info/gems/administrate/Administrate/Field/Select
+%>
+
+<%= field.data.presence %>
```

Comment:
Is the `.presence` necessary? I would expect it to behave the same without it, because the result is getting rendered as a string either way.